tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-confusion-matrix.html

```diff
@@ -0,0 +1,256 @@
+<!--
+@license
+Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<link rel="import" href="../polymer/polymer.html">
+
+<!-- Displays a simple confusion matrix. -->
+<dom-module id="tf-confusion-matrix">
+  <template>
+    <style>
+      table {
+        display: inline-block;
+      }
+      td {
+        text-align: right;
+      }
+      th, td {
+        padding: 3px;
+        font-weight: normal;
+      }
+      .flex {
+        display: flex;
+        flex-direction: row-reverse;
+      }
+      .pred-label {
+        text-align: left;
+        font-size: 10px;
+      }
+      .actual-label {
+        text-align: right;
+        font-size: 10px;
+      }
+      .n-label {
+        font-size: 10px;
+        vertical-align: bottom;
+      }
+      .correct {
+        border-left: 3px solid DarkSeaGreen;
+      }
+      .incorrect {
+        border-left: 3px solid lightcoral;
+      }
+      .blank {
+        background: darkgray;
+      }
+      .parenthetical {
+        padding-right: 5px;
+        color: gray;
+        width: 50px;
+      }
+    </style>
+    <div id="holder"></div>
+  </template>
+
+  <script>
+    "use strict";
+
+    Polymer({
+      is: "tf-confusion-matrix",
+      properties: {
+        counts: {type: Object, observer: 'drawMatrix'},
```

Comment:
Can you please document the structure of the `counts` either with jsdoc above or an example in a comment? Thanks!

Reply:
done